sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/importer/CampaignFormDataImporter.java

```diff
@@ -213,4 +238,20 @@ private CampaignFormDataDto insertColumnEntryIntoData(CampaignFormDataDto campai
 		}
 		return currentElement;
 	}
+
+	@Override
+	protected boolean executeDefaultInvokings(PropertyDescriptor pd, Object element, String entry, String[] entryHeaderPath)
+		throws InvocationTargetException, IllegalAccessException, ParseException, ImportErrorException {
+		final boolean returnBoolean = super.executeDefaultInvokings(pd, element, entry, entryHeaderPath);
```

Comment:
If you override executeDefaultInvokings anyway (without calling super), you could also get rid of the call in the above method and just add the region code there

Reply:
what i want to achieve here is to execute the jurisdiction check for region before the import get's to parse the district
super is called - and if no exception is thrown i return the value from super here

Reply:
Okay, I see - fine by me then :)